### ISSUE TITLE
[Windows] odd background color of icons in company candidates menu

### ISSUE THREAD

**Issue by grolongo** [protesilaos/modus-themes#61]
Hi Prot!  
Since I updated modus-theme from v3 to v4, there is an odd background being applied on icons in the company candidates popup.  
To be more clear here are two screenshots using modus-operandi and the new modus-operandi-tinted (which I love btw):  

modus-operandi v4
![operandi-v4](https://user-images.githubusercontent.com/34292770/211395326-d1783ded-64e3-44e5-87be-b10bd5c4a572.png)

modus-operandi-tinted v4
![operandi-inted-v4](https://user-images.githubusercontent.com/34292770/211395492-b737dc35-2531-4f23-863b-012301e58c3e.png)

Curiously, I'm using the same `init.el` across my machines and the problem only appears on Windows (using Emacs 28.2), no problem on macOS or Linux.
I don't think you have a Windows machine to test things out so feel free to ask me whatever info you need.

Have a good evening!

**Comment by protesilaos** [comment]
Hello @grolongo!  Indeed, I do not have a Windows machine, but I will try to help you.

Do you know which package produces those icons?  Are they "icon files" or "font files" (like FontAwesome)?  I am asking because this may not be a theme issue but something related to font rendering.

Is this problem limited to the Modus themes?

**Comment by grolongo** [comment]
Thanks for the reply,

>Is this problem limited to the Modus themes?

So I was indeed mistaken since I can reproduce the error using other built-in themes, so modus-themes is not the problem.  
I observe the same behavior using the default theme, leuven and adwaita. It is looking fine though when using wombat, tango-dark or wheatgrass... this is strange.

>Do you know which package produces those icons? Are they "icon files" or "font files" (like FontAwesome)?

These are transparent svg icon files automatically pulled by Company, you can find them there: https://github.com/company-mode/company-mode/tree/master/icons

Feel free to close the issue since it looks like it is out of scope of modus-themes unless there is something I can tweak? I'll keep investigate.

**Comment by protesilaos** [comment]
Let's keep this open, as other users may also encounter the problem.  If you find a solution, please post it here.  We can close the issue afterwards.

**Comment by grolongo** [comment]
Hi again Prot!

So I know this issue affects other built-in themes and is not related solely to modus-themes, but I did some testing and maybe you can help me pinpoint the problem further.  

Using `Emacs 28.2` from the official website, this is the bare bone `init.el` I'm using (no `early-init.el` involved):
```elisp
;; package.el and MELPA
(require 'package)
(add-to-list 'package-archives '("melpa" . "https://melpa.org/packages/") t)

;; Make sure `use-package’ is available.
(unless (package-installed-p 'use-package)
  (package-refresh-contents)
  (package-install 'use-package))

;; Configure `use-package' prior to loading it.
(eval-and-compile
  (setq use-package-always-demand t))

(eval-when-compile
  (require 'use-package))

(use-package company
  :ensure t
  :config (global-company-mode 1))

(use-package modus-themes
  :pin gnu
  :ensure t)
```

By using `:pin gnu` I'm getting `modus-themes` from Elpa (v3.0), and the _correct_ behavior is shown:

![modus-v3](https://user-images.githubusercontent.com/34292770/211881546-4dcc6bd5-8249-4cac-b52c-9b512bb27fd4.png)

Now if I remove `:pin gnu` I'm getting the latest `modus-themes` from Melpa (v4.0+), it produces the background issue:

![modus-v4](https://user-images.githubusercontent.com/34292770/211882088-aa9eea5b-cb41-4343-9542-2356481ea41e.png)

Do you have an idea of what might have changed?

**Comment by protesilaos** [comment]
How do you enable the icons?  Enabling `company-mode` does not do it for me.  I will try to find what is happening.

**Comment by protesilaos** [comment]
By the way, the background of the popup should be lighter.  I corrected it just now.

**Comment by grolongo** [comment]
>On Arch Linux I get the expected results  

This is what is driving me nuts haha. I have normal behavior on my Linux and macOS machine but only on Windows it is doing this even though it's the same config. This is so confusing.

I changed the line manually `bg-inactive` to `bg-dim` as in your latest commit [5ce2bd34](https://git.sr.ht/~protesilaos/modus-themes/commit/5ce2bd34b1b97e9776d9073783dc2840672038de) but no luck:

![modus-patched](https://user-images.githubusercontent.com/34292770/212047956-394fc64c-3b64-495f-a891-4cd5dd45f491.png)

At this point it might be a deeper bug into libsvg on Windows and I might report it as @dgutov suggested.

**Comment by protesilaos** [comment]
> At this point it might be a deeper bug into libsvg on Windows

It could be.  It still is confusing why Modus 3 works but 4 doesn't.  It must mean that a change to another face is impacting this.  But which one?

Using the inspector of Firefox, I get that the background colour of those icons is `#ffdec0`.  I cannot find this in the Modus' repo.  Same for the `#cfd5df` that appears in your "tinted" screenshot.

**Comment by grolongo** [comment]
Now I'm not sure if the jpeg compression of the screenshot alters it or if a bug is replacing those values.

**Comment by protesilaos** [comment]
Yes, that's the other problem we have.  I searched the `M-x list-faces-display` in hope of finding something but could not narrow it down.

**Comment by dgutov** [comment]
> Now I'm not sure if the jpeg compression of the screenshot alters it or if a bug is replacing those values

You could save the screenshot as png.

**Comment by dgutov** [comment]
The behavior on Windows differing from Linux and Mac makes it almost certainly a Windows bug.

It would be nice to find a quick workaround, but we hadn't managed so far. The rest of the investigation should probably consist of gathering better data for the bug report. But we probably have enough of it already -- even the quick snippet to reproduce the problem.

**Comment by grolongo** [comment]
>You could save the screenshot as png.

My bad, the screenshot is already in png, not jpg.

>The rest of the investigation should probably consist of gathering better data for the bug report.

Before going any further, it would be "nice" to find at least one other person using Emacs on Windows reproducing the same bug with the minimal `init.el`, to make sure it's not just my system that is doing this for some reason.  

I haven't seen anybody else reporting a similar issue or jumping into this bug report. Maybe that's too trivial for them to even care which is fair, or maybe I'm just the only one.

**Comment by dgutov** [comment]
The fact of the matter is, not everybody who encounters the problem, reports it. Alas.
